packages/charts/react-charting/src/types/IDataPoint.ts

```diff
@@ -964,3 +964,25 @@ export interface IScatterChartPoints {
    */
   useSecondaryYScale?: boolean;
 }
+
+/**
+ * {@docCategory CartesianChart}
+ */
+export type AxisCategoryOrder =
```

Comment:
can you a add a 1 line description about the meaning of each of these options at the top

Reply:
done